packages/plugin-table/src/TableNode.ts

```diff
@@ -1,7 +1,81 @@
 import { VElement } from '../../core/src/VNodes/VElement';
+import { TableRowNode } from './TableRowNode';
+import { TableCellNode } from './TableCellNode';
 
 export class TableNode extends VElement {
     constructor() {
         super('TABLE');
     }
+
+    //--------------------------------------------------------------------------
+    // Getters
+    //--------------------------------------------------------------------------
+
+    /**
+     * @override
+     */
+    get name(): string {
+        return super.name + ': ' + this.rowCount + 'x' + this.columnCount;
+    }
+    /**
+     * Return an array of row nodes in this table.
+     */
+    get rowNodes(): TableRowNode[] {
+        return this.children(TableRowNode);
+    }
+    /**
+     * Return an array of rows in this table, as arrays of cells.
+     */
+    get rows(): TableCellNode[][] {
+        const rows = [];
+        for (const row of this.rowNodes) {
+            const rowGrid = [];
+            for (let columnIndex = 0; columnIndex < this.columnCount; columnIndex += 1) {
+                rowGrid.push(this.rowNodes[row.rowIndex].cells[columnIndex]);
+            }
```

Comment:
Why something like `rowGrid.push([...this.rowNodes[row.rowIndex].cells])` wouldn't work ?

Reply:
I just realized you did not force push the branch, did you ?